pages/common/assets/sass/style.scss

```diff
@@ -1,4 +1,7 @@
+$govuk-font-family: 'Inter UI', Helvetica, Arial, sans-serif;
```

Comment:
It is set in `frontend-toolkit` as an overridable default (needs the extra fallbacks adding):
https://github.com/UKHomeOffice/frontend-toolkit/blob/master/assets/styles/_typography.scss#L1

But... we need to set the var before we include any `govuk-frontend` components, otherwise they will bring in the NTA font-face declarations when compiled, which means either 404s or loading fonts that we don't use.

We could change the ordering of:
https://github.com/UKHomeOffice/asl-pages/blob/master/pages/common/assets/sass/_govuk.scss

...to bring in `@ukhomeoffice/frontend-toolkit` before the `govuk-frontend` components, but then you won't be able to override any component styling with it, so that's why it's the order it currently is.

Thoughts?

Reply:
Is that not just a question of re-ordering the following then?

```
@import "govuk";
@import "default"
```

Reply:
Nope, it's the order in `_govuk.scss`:

```
@import "@ukhomeoffice/frontend-toolkit";
// then
@import "govuk-frontend/components/breadcrumbs/breadcrumbs";
@import "govuk-frontend/components/error-summary/error-summary";
@import "govuk-frontend/components/phase-banner/phase-banner";
```
but that would mean we can't use frontend-toolkit to override the component styling.